include/ur_api.h

```diff
@@ -1697,6 +1699,7 @@ typedef enum ur_device_info_t {
                                                                      ///< backed 2D sampled image data.
     UR_DEVICE_INFO_ENQUEUE_NATIVE_COMMAND_SUPPORT_EXP = 0x2020,      ///< [::ur_bool_t] returns true if the device supports enqueueing of native
                                                                      ///< work
+    UR_DEVICE_INFO_LOW_POWER_EVENTS_EXP = 0x2021,                    ///< [::ur_bool_t] returns true if the device supports low-power events.
```

Comment:
I am not against having this query, but it is not strictly necessary, given the property is a hint, so backends can just ignore it. No harm in it though!

Reply:
I added it mostly for consistency sake. All the other experimental extensions add a query like that so 🤷